subsys/bluetooth/controller/ll_sw/lll.h

```diff
@@ -157,6 +157,8 @@ struct lll_event {
 enum node_rx_type {
 	/* Unused */
 	NODE_RX_TYPE_NONE = 0x00,
+	/* Signals simple release of node */
+	NODE_RX_TYPE_RELEASE,
```

Comment:
That type is used only in ULL, it's never sent from LLL. I can add add handling so it can be also sent from LLL, but perhaps we should also rename it to `NODE_RX_TYPE_RELEASE` since it will now release non-DC PDUs also?